README.rst

```diff
@@ -100,13 +101,34 @@ Dependencies
 
 `dateparser` translates non-English dates to English and uses dateutil_ module ``parser`` to parse the translated date.
 
-Also, it requires PyYAML_ for its language detection module to work.
+Also, it requires PyYAML_ for its language detection module to work. The module jdatetime_ is used for handling Jalali calendar.
 
 .. _dateutil: https://pypi.python.org/pypi/python-dateutil
 .. _PyYAML: https://pypi.python.org/pypi/PyYAML
+.. _jdatetime: https://pypi.python.org/pypi/jdatetime
 
 
-Limitations
+Supported languages
```

Comment:
May be we can add a section `Supported Calendars` and add JalaliParser usage examples there.